### ISSUE TITLE
Broken SRV lookup (DNS records)

### ISSUE THREAD

**Issue by MattCowley** [ORelio/Minecraft-Console-Client#253]
So I'm trying to connect to a 1.11.2 using the latest exe from https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts

However I get this error:
https://gyazo.com/61537dfd4a383c57e7e8e18a8255a32d

**Comment by ORelio** [comment]
The specified IP address is probably wrong or missing port number.

**Comment by MattCowley** [comment]
@ORelio The IP is zerpau.com
I can connect to it via a normal minecraft client but not via the console client. Any ideas?

**Comment by ORelio** [comment]
Seems like there are issue with load balanced IP addresses i.e. when a domain has several IP:
````
$ host zerpau.com
zerpau.com has address 107.154.249.52
zerpau.com has address 107.154.248.52
``````
Will dig this later, thanks for your report :)

**Comment by MattCowley** [comment]
It's an enjin server if that helps at all. :)

**Comment by ORelio** [comment]
After investigating, in fact, load balanced domains works well out of the box, the issue was with SRV lookups, the [`DnDns`](https://dndns.codeplex.com/) library included in MCC wasn't doing a great job at resolving them.

SRV records are sometimes used to redirect Minecraft to the actual server domain name and port, and for `zerpau.com` the SRV record is `zerpau.com.zerpau.com:25575` which resolves to `103.231.91.71:25575`, and MCC failed to do that, so it defaulted to resolving `zerpau.com` to either `107.154.249.52:25565` or `107.154.248.52:25565` (not the correct server IP address and port).

SRV lookup is now rewritten in MCC using [`HeinjenDns`](https://github.com/ghuntley/Heijden.Dns) library instead, seems like that library works way better at resolving SRV records, this should fix your issue. You can try it out in latest [development build](https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts) :)

**Comment by MattCowley** [comment]
Oh cool, I'm glad you got it sorted out. Working perfectly. Thank you very much!
